disnake/ext/commands/params.py

```diff
@@ -321,27 +401,6 @@ async def get_default(self, inter: CommandInteraction) -> Any:
 
         return default
 
-    async def verify_type(self, inter: CommandInteraction, argument: Any) -> Any:
-        """Check if a type of an argument is correct and possibly fix it"""
-        # these types never need to be verified
-        if self.discord_type.value in [3, 4, 5, 8, 9, 10]:
-            return argument
-
-        if issubclass(self.type, disnake.Member):
```

Comment:
Looks like we actually shouldn't remove this, since Discord doesn't differentiate between user and member parameters - without it, you could annotate a param with `Member`, pass a non-member ID and `argument` would be a `User`.
I originally thought this change would be fine, but I just ran into this issue

Not quite sure about the `GuildChannel` part, but the `Member` part should stay